protocol/communities/manager.go

```diff
@@ -1789,19 +1797,26 @@ func (m *Manager) HandleCommunityDescriptionMessage(signer *ecdsa.PublicKey, des
 	if err != nil {
 		return nil, err
 	}
-	m.communityLock.Lock(id)
-	defer m.communityLock.Unlock(id)
+
 	community, err := m.GetByID(id)
 	if err != nil && err != ErrOrgNotFound {
 		return nil, err
 	}
 
+	// We should not process the description from ourselves if we are the control node
+	if community != nil && community.IsControlNode() && common.IsPubKeyEqual(community.ControlNode(), signer) {
```

Comment:
```suggestion
	if community != nil && community.IsControlNode() {
```
Should be enough.

Reply:
Not sure if the token master or another administrator updates the description? I think in this case the description should be handled by the management node, so checking signer of received message make sense for me

Reply:
There can be only one control node (one device) at a time.